cfdm/core/domaintopology.py

```diff
@@ -0,0 +1,302 @@
+from . import abstract
+
+
+class DomainTopology(abstract.Topology):
+    """A domain topology construct of the CF data model.
+
+    A domain topology construct defines the geospatial topology of
+    cells arranged in two or three dimensions in real space but
+    indexed by a single (discrete) domain axis construct, and at most
+    one domain topology construct may be associated with any such
+    domain axis. The topology describes topological relationships
+    between the cells - spatial relationships which do not depend on
+    the cell locations - and is represented by an undirected graph,
+    i.e. a mesh in which pairs of nodes are connected by links. Each
```

Comment:
Hi Sadie, this text is lifted verbatim from the data model description in CF Appendix I, so if it were to be changed then it should be changed there first. But what is the confusion, here?

Reply:
Ah, fair. In which case we do want to keep them matched up. If the following doesn't seem like an issue to you, no worries, since it could just be me anticipating some minor confusion based on my having to double check something from elsewhere in the text (could just be a 'me' thing)...

Specifically here I was concerned that 'a mesh in which pairs of nodes _are_ connected by links' could imply that _all_ nodes must be connected to _all_ other nodes. I realise now the word I forgot to describe this mathematically, which is a [_complete graph_](https://en.wikipedia.org/wiki/Complete_graph). Which from the other description and UGRID context doesn't seem to be the case - nodes might need to be connected to at least one other so as to not make the graph disconnected (I think?), but otherwise no requirement on how many links should be present for a given node, right?

Hence my suggestion of a small tweak to the wording, but if it seems clear enough w.r.t. the above to you, then we can leave it. We'd have to change it in two locations after all, as you point out.

Reply:
Your graph theory observations are spot on. It might have been good to use the word "complete", but that's one to consider over at CF, if we want to (but not before 1.11 is released!).